src/fastga/models/aerodynamics/external/openvsp/openvsp.py

```diff
@@ -457,6 +457,9 @@ def compute_wing(self, inputs, outputs, altitude, mach, aoa_angle):
         output_file_list = [
             pth.join(target_directory, INPUT_WING_SCRIPT.replace(".vspscript", "_DegenGeom.csv"))
         ]
+
+        # This requires the installation of pyparsing and consequently adding it to the
+        # pyproject.toml, we may be able to do without it (TO-DO ?).
```

Comment:
To avoid any confusion I would remove this comment since apparently this is fixed the present PR.

Reply:
Done 😄